### DISCUSSION TITLE
Can I use the elements in versions.json to represent aliases for folder names?

### DISCUSSION THREAD

**Discussion by ms2art** [jimporter/mike#182] [General]
I'm trying to create a version selector for the folders in which the two versions of my documentation are stored in Git. The folder naming isn't consistent, for historical reasons, so I'd like to specify the actual path/folder names and provide aliases which will appear in the version selector.  Specifically, my first version is stored in Git in:

<repo_name>
    docs
        docs
            documentation

and my latest is in:

<repo_name>
    docs
        95
            docs
                documentation

I'd like "9.2" to be the text in the dropdown that takes the user to "docs/documentation/index.html", and "9.5" to display 95/docs/documentation/index.html.

Is this possible with mike and versions.json?

**Comment by jimporter** [comment]
You can set titles for each version to anything you like, which would help with the naming. However, you'll have to move your documentation around, since mike requires each version to be siblings in the directory structure. (This eliminates the risk of one doc version clobbering another version's pages.)

So ultimately, the answer is no, it isn't possible to keep that directory structure.

**Reply by ms2art** [reply]
That's too bad. But I appreciate your quick reply!
